### ISSUE TITLE
Limit zooming out pdf Scale

### ISSUE THREAD

**Issue by tinywhui** [mozilla/pdf.js#8362]
Link to PDF file (or attach file here):

Configuration:
- Web browser and its version: Chrome
- Operating system and its version: WIN10
- PDF.js version:
- Is an extension:

Steps to reproduce the problem:
1. In Viewer.html, I hide option 0.5 and it will not display.
2. Set a min value to 0.5 of CustomScaleOption.

What is the expected behavior? (add screenshot)
Cannot zoom out less than 50%
What went wrong? (add screenshot)
It does not work, you can still zoom out until 25%
Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

The default minimum displaying scale for pdf file is 25%, how can we change it to other value? For example 50%.

**Comment by yurydelendik** [comment]
It's hard to follow provided steps. Please provide exact modifications in form of diff.

**Comment by tinywhui** [comment]
<option title="" value="0.5" disabled="disabled" hidden="true" data-l10n-id="page_scale_percent" data-l10n-args='{ "scale": 50 }'>50%</option>
I added a hidden figure there so that it will not display the scale of 50%. However, I can still use Zoom in and Zoom out to reach a scale with 50%. I noticed the default minimum scale is 25%, but I did not find if the default value can be modified in the source code or not. If so, how can the scale be set to a minimum value, let's say, 50% instead of the 25% default value?

**Comment by tinywhui** [comment]
in web/viewer.html, under id="scaleSelectContainer" class="dropdownToolbarButton"
title="" value="0.5" disabled="disabled" **_hidden="true"_** data-l10n-id="page_scale_percent" data-l10n-args='{ "scale": 50 }'>50%

 I added a hidden figure there so that it will not display the scale of 50%. However, I can still use Zoom in and Zoom out to reach a scale with 50%. I noticed the default minimum scale is 25%, but I did not find if the default value can be modified in the source code or not. If so, how can the scale be set to a minimum value, let's say, 50% instead of the 25% default value?

**Comment by Snuffleupagus** [comment]
> If so, how can the scale be set to a minimum value, let's say, 50% instead of the 25% default value?

By changing `MIN_SCALE` in https://github.com/mozilla/pdf.js/blob/aea3eccd0ff8601b72396f74ba48f759f72a6886/web/ui_utils.js#L21, closing as answered.
